src/test/scala/uk/gov/hmrc/utils/TestConfiguration.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.integration.cucumber.utils
+
+import com.typesafe.config.ConfigFactory
+
+object TestConfiguration {
+
+  val timeout                    = 30
+  val env: String                = Option(System.getProperty("environment")).getOrElse("local")
+  val javascriptDisabled: String = Option(System.getProperty("disable.javascript")).getOrElse("false")
+  private val config             = ConfigFactory.load("environments.conf").getConfig(s"environments.$env")
+
+  def baseUrl(url: String, envHostForEnv: String = environmentHost): String =
+    if (env.equalsIgnoreCase("local")) s"$environmentHost:${servicePort(url)}${serviceRoute(url)}"
+    else s"$envHostForEnv${serviceRoute(url)}"
+
+  def url(url: String): String =
+    url match {
+      case "UsingThisServiceUrl"             => baseUrl("dds") + "using-this-service"
```

Comment:
Apologies for the late comment but I've been having a chat with Mike regarding his current AMLS AT PR and we think that we can make a change to this sort of String matching. Essentially we want to remove the need to have "Url" suffix on the end of these. Can you reach out to him to understand what he's going to do for this so we can keep it consistent.